content/docs/operating/configuration.md

```diff
@@ -114,11 +128,25 @@ job_name: <name>
 # The URL scheme with which to fetch metrics from targets.
 [ scheme: <scheme> | default = http ]
 
-# HTTP basic authentication information.
+# Optional authentication information. Note that basic_auth, bearer_token &
```

Comment:
Please put backticks around `basic_auth`, `bearer_token`, ... to format them as code.